### PR TITLE
Update EIP-918: Move from Stagnant to Draft

### PR DESCRIPTION
**Pull request by jlogelin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by mryalamanchi** [comment]
@jlogelin kindly add the github username for atleast one of the author, instead of the e-mail id.

**Review by lightclient — CHANGES_REQUESTED** [review]
@jlogelin please review EIP-1 and the EIP template. The format has changed since this EIP was created. It will need to be updated before moving it to last call.

**Comment by jlogelin** [comment]
---
## (fail) EIPS/eip-918.md
| classification |
| ------------- |
| `ambiguous` |
- eip-918.md has no identifiable authors who can approve the PR (only considering the base version)
Error: Process completed with exit code 1.

@mryalamanchi please advise

File: EIPS/eip-918.md [hunk 1]
```diff
@@ -1,27 +1,21 @@
 ---
 eip: 918
 title: Mineable Token Standard
+description: A specification for standardized Mineable Tokens that uses a Proof of Work algorithm for distribution.
 author: Jay Logelin (@jlogelin), Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
-type: Standards Track
-category: ERC
 status: Last Call
+type: Standards Track
+category ERC
```
Comment (lightclient) [inline EIPS/eip-918.md]:
```suggestion
category: ERC
```

**Review by lightclient — CHANGES_REQUESTED** [review]

**Review by jlogelin — COMMENTED** [review]
Looks good

File: EIPS/eip-918.md [hunk 2]
```diff
@@ -1,27 +1,21 @@
 ---
 eip: 918
 title: Mineable Token Standard
+description: A specification for standardized Mineable Tokens that uses a Proof of Work algorithm for distribution.
 author: Jay Logelin <jlogelin@alumni.harvard.edu>, Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
```
Comment (MicahZoltu) [inline EIPS/eip-918.md:5]: Something odd is happening with this PR.  I see the latest commit that added a GitHub user to the author list, but I don't see that reflected here:
image
Maybe this PR needs a rebase or cleanup or something?

Comment (jlogelin) [reply EIPS/eip-918.md]: Rebased and changed username to `jlogelin` in the header. Still seeing the same errors, Please advise @MicahZoltu

Comment (jlogelin) [reply EIPS/eip-918.md]: Rebased, changed username in header and still seeing the same error. Please advise @MicahZoltu 

```
---
## (fail) EIPS/eip-918.md
| classification |
| ------------- |
| `ambiguous` |
- eip-918.md has no identifiable authors who can approve the PR (only considering the base version)
Error: Process completed with exit code 1.
```

**Comment by MicahZoltu** [comment]
> ./EIPS/eip-918.md:473: explicitely ==> explicitly

**Comment by MicahZoltu** [comment]
I verified that the author change is legitimate.  You'll need to manually merge once the above spelling error is fixed and CI passes @lightclient.

File: EIPS/eip-918.md [hunk 3]
```diff
@@ -1,27 +1,21 @@
 ---
 eip: 918
 title: Mineable Token Standard
-author: Jay Logelin <jlogelin@alumni.harvard.edu>, Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
+description: A specification for standardized Mineable Tokens that uses a Proof of Work algorithm for distribution.
+author: Jay Logelin (@jlogelin), Infernal_toast <admin@0xbitcoin.org>, Michael Seiler <mgs33@cornell.edu>, Brandon Grill <bg2655@columbia.edu>
+status: Last Call
```
Comment (lightclient) [inline EIPS/eip-918.md:6]:
```suggestion
status: Draft
```
This was the last status of the EIP before being moved to stagnant, so it should return to draft before moving to last call.

**Review by lightclient — CHANGES_REQUESTED** [review]

**Pull request closed without merging**